tests/integration/pom.xml

```diff
@@ -132,6 +132,8 @@
         <!-- TODO: conflict in ASM version (too old) of jetty plugin -->
         <!--<module>spring4</module>-->
         <module>tracing-support</module>
+        <module>microprofile/config/helidon</module>
```

Comment:
Why is it here? Should it be in ext/microprofile?

Reply:
it was previously discussed (25.4.2019) and the decision was to extract that from ext/microprofile/config to tests. In ext/microprofile/config we shall have only dependency to Eclipse's microprofile API. And all direct implementations (small rye, helidon) shall be put into tests.